test/integration/functional_test.go

```diff
@@ -583,6 +586,18 @@ func validateDashboardCmd(ctx context.Context, t *testing.T, profile string) {
 	}
 }
 
+// dashboardURL gets the dashboard URL from the command stdout.
+func dashboardURL(b *bufio.Reader) (string, error) {
+	s := bufio.NewScanner(b)
+	for s.Scan() {
+		l := s.Text()
+		if strings.HasPrefix(l, "http") {
```

Comment:
how about making it more strict that has both http and the dashboard so it does not match any random HTTP word in the stdout ?